var/spack/repos/builtin/packages/py-pyvista/package.py

```diff
@@ -12,6 +12,9 @@ class PyPyvista(PythonPackage):
     homepage = "https://github.com/pyvista/pyvista"
     pypi = "pyvista/pyvista-0.32.1.tar.gz"
 
+    # Requires optional trame dependency
+    skip_modules = ["pyvista.trame"]
+
```

Comment:
I'd maybe also skip `pyvista.ext`

Reply:
And maybe also `pyvista.jupyter`, though this is not needed now. Future versions of PyVista may do things in that module that require jupyter or trame dependencies.